### PR TITLE
MCR-1622 change date modified when embargo ends

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-1622).

### CODE REVIEW

File: mycore-mods/src/main/java/org/mycore/mods/MCRMODSEmbargoCronjob.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    private void releaseDocument(MCRObjectID objectID) {
+        final MCRSession session = MCRSessionMgr.getCurrentSession();
+        try {
+            LOGGER.info("Release object {}", objectID);
+            session.setUserInformation(MCRSystemUserInformation.getSuperUserInstance());
+            final MCRObject object = MCRMetadataManager.retrieveMCRObject(objectID);
+            final MCRMODSWrapper modsWrapper = new MCRMODSWrapper(object);
+            final String embargoXPATH = "mods:accessCondition[@type='embargo']";
+            final String embargoString = modsWrapper.getElement(embargoXPATH).getTextNormalize();
```
Comment (kkrebs) [inline mycore-mods/src/main/java/org/mycore/mods/MCRMODSEmbargoCronjob.java]: get an NPE while application startup here

```
[INFO] [talledLocalContainer] Exception in thread "Timer-0" java.lang.NullPointerException
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.releaseDocument(MCRMODSEmbargoCronjob.java:101)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[INFO] [talledLocalContainer]   at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1380)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.searchDocumentsToRelease(MCRMODSEmbargoCronjob.java:87)
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.run(MCRMODSEmbargoCronjob.java:117)
[INFO] [talledLocalContainer]   at java.util.TimerThread.mainLoop(Timer.java:555)
[INFO] [talledLocalContainer]   at java.util.TimerThread.run(Timer.java:505)
```

**Review by kkrebs — CHANGES_REQUESTED** [review]
see comment

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 3f4f61e771d4355c8e67e5a82ce6b493cab1d72c! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/3225-pull_request-572

**Review by kkrebs — COMMENTED** [review]
I still get an Exception on startup

[INFO] [talledLocalContainer] 2018-05-16T10:58:33,555 INFO   MCRStoreManager: Adding instance of MCRVersioningMetadataStore as MCRStore 'oer_derivate'.
[INFO] [talledLocalContainer] Exception in thread "Timer-0" java.lang.NullPointerException
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.releaseDocument(MCRMODSEmbargoCronjob.java:101)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
[INFO] [talledLocalContainer]   at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1380)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
[INFO] [talledLocalContainer]   at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
[INFO] [talledLocalContainer]   at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[INFO] [talledLocalContainer]   at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.searchDocumentsToRelease(MCRMODSEmbargoCronjob.java:87)
[INFO] [talledLocalContainer]   at org.mycore.mods.MCRMODSEmbargoCronjob.run(MCRMODSEmbargoCronjob.java:117)
[INFO] [talledLocalContainer]   at java.util.TimerThread.mainLoop(Timer.java:555)
[INFO] [talledLocalContainer]   at java.util.TimerThread.run(Timer.java:505)

**Review by kkrebs — APPROVED** [review]

**Pull request merged**